support/cas-server-support-themes-collection/src/main/resources/templates/twbs/layout.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout">
+
+<head>
```

Comment:
Same question. You should not include this file in the theme. Are you customizing this only to remove core css/js libraries from the layout? If so, you should use a theme setting for this that is already available.